.github/workflows/wheels.yml

```diff
@@ -130,6 +130,10 @@ jobs:
       - name: Setup macOS
         if: matrix.buildplat[0] == 'macos-13' || matrix.buildplat[0] == 'macos-14'
         run: |
+          # Needed due to https://github.com/actions/runner-images/issues/3371
+          echo "FC=gfortran-10" >> "$GITHUB_ENV"
+          echo "F77=gfortran-10" >> "$GITHUB_ENV"
+          echo "F90=gfortran-10" >> "$GITHUB_ENV"
```

Comment:
It looks like the images no longer have `gfortran-10` except for `macos-11`, which we don't use. All of the ones we use have 11, 12, and 13 installed. https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md

Reply:
Yikes, my bad, should be fine now.